scm/src/gmtb_scm.F90

```diff
@@ -297,12 +323,18 @@ subroutine gmtb_scm_main_sub()
     end if
 
     if(mod(scm_state%itt, scm_state%n_itt_out)==0) then
-      scm_state%itt_out = scm_state%itt_out+1
+      
       write(*,*) "itt = ",scm_state%itt
       write(*,*) "model time (s) = ",scm_state%model_time
+      write(*,*) "Bowen ratio: ",physics%Interstitial%dtsfc1(1)/physics%Interstitial%dqsfc1(1)
```

Comment:
This seems like leftover code from debugging the code? Do you want to output Bowen ratio etc. for all kinds of SCM runs and every time step? Even for runs that don't use the LSM capability?

Reply:
This was wrapped in an if statement checking for lsm_ics/model_ics at some point. I think it got omitted when I merged in some older code. I'll re-add the conditional since it is really nice for debugging/tuning soil moisture when using lsm_ics.